migrations/20200828113656_migration_add_status_for_tickets.ts

```diff
@@ -0,0 +1,27 @@
+import * as Knex from 'knex'
+
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.table('tickets', table => {
+    table.enu('status',
```

Comment:
I guess this should be `table.enum`

Reply:
"Adds a enum column, (aliased to enu, as enum is a reserved word in JavaScript)."